Apps/AdminWebClient/src/ClientApp/src/constants/resulttype.ts

```diff
@@ -1,4 +1,4 @@
-export enum ResultType {
+export const enum ResultType {
```

Comment:
Why are you changing all of the enums to constants?  This applies to multiple types below.  I disagree with the change as it allows for errors in the code.  Please convert back and then add value convertors as required in the DB Context (we have examples of that).

Reply:
they are still enum's, only const enum's now :)